app/javascript/components/shared/helpers.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const preserveLines = string => `${string}`.split("\n").map((line, idx) => <p key={idx}>{line}</p>);
+export const unnullifyFields = (object) => {
+  const result = Object.create(null);
+  for (let field in object) {
+    if (object[field] === null) {
```

Comment:
just use `field` to compare it with null

Reply:
I do not want to replace 0 with ''